Filter/ModelFilter.php

```diff
@@ -113,6 +114,29 @@ protected function handleMultiple(ProxyQueryInterface $queryBuilder, $alias, $da
     }
 
     /**
+     * @param ProxyQueryInterface $queryBuilder
```

Comment:
Please describe what this does. IMO it returns the alias of the parent entity if there is one, right?